src/natcap/invest/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 LOGGER = logging.getLogger('natcap.invest')
 LOGGER.addHandler(logging.NullHandler())
-__all__ = ['local_dir',]
+__all__ = ['local_dir', ]
```

Comment:
This is trivial, but do we need that trailing comma and space at all?

Reply:
I think I added that when I was trying to do some pycodestyle cleaning. I believe it recommended the space after comma.